### ISSUE TITLE
Support for the filter 

### ISSUE THREAD

**Issue by SIGEER2222** [KristofferStrube/Blazor.SVGEditor#34]
I noticed that support for the filter element is not added in the code. Is there a plan to do it in the future?

**Comment by KristofferStrube** [comment]
Yeah, definitely. I can't say when. If you have a need for it I might be able to prioritize it. :)
